app.py

```diff
@@ -0,0 +1,61 @@
+import json
```

Comment:
Place the app in its own folder in the repo... Treat the folder like it was it's own repo. It will container the Dockerfile and all other files. You'll need to update the docker-compose.yml file context to look in the folder for the dockerfile.